packages/manager/src/queries/cloudpulse/tags.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { queryFactory } from './queries';
+
+import type { APIError, Filter, Params } from '@linode/api-v4';
+import type { CloudPulseTags } from 'src/features/CloudPulse/shared/CloudPulseTagsFilter';
+
+export const useTagsQuery = (
+  enabled = false,
+  resourceType: string | undefined,
+  params?: Params,
+  filters?: Filter
+) =>
+  useQuery<any[], APIError[], CloudPulseTags[]>({
```

Comment:
Here it will always be linode type for tags...we can use the linode instance all query directly from their query factory and have it like 
```suggestion
  useQuery<Linode[], APIError[], CloudPulseTags[]>({
```